release/schedule/index.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: Release Schedule
+---
+
+WPE WebKit follows a 6-month development cycle, where:
```

Comment:
I think it's less awkward to write simply:

"WPE WebKit follows a 6-month development cycle:"